packages/entrykit/src/useAppChain.tsx

```diff
@@ -0,0 +1,16 @@
+import { useChains } from "wagmi";
+import { MUDChain } from "@latticexyz/common/chains";
+import { useEntryKitConfig } from "./EntryKitConfigProvider";
+
+export function useAppChain(): MUDChain {
+  const { chainId } = useEntryKitConfig();
```

Comment:
this hook is unnecessary as `useEntryKitConfig()` already includes `chain` object